package.json

```diff
@@ -168,21 +168,26 @@
         }
     },
     "scripts": {
-        "vscode:prepublish": "npm run compile",
+        "vscode:prepublish": "npm run tslint-check && npm run compile",
         "compile": "tsc -p ./",
         "watch": "tsc -watch -p ./",
-        "postinstall": "node ./node_modules/vscode/bin/install"
+        "postinstall": "node ./node_modules/vscode/bin/install",
+        "tslint-check": "tslint -c ./tslint.json ./src/**/*.ts",
+        "format": "prettier --write './src/**/*.ts'"
     },
     "devDependencies": {
+        "@types/node": "^10.0.3",
+        "prettier": "^1.13.7",
+        "tslint": "^5.10.0",
+        "tslint-plugin-prettier": "^1.3.0",
         "typescript": "^2.8.3",
-        "vscode": "^1.1.6",
-        "@types/node": "^10.0.3"
+        "vscode": "^1.1.6"
     },
     "dependencies": {
+        "@octokit/rest": "^15.9.4",
```

Comment:
It was in my backlog, as you have changed it, do make sure it dont break existing functions

Reply:
yes. sure!

After refactor done, I will try all function one by one.

If possible, I will write unit tests for those service.